brendenwest/cascade
# Setup main menu

## Description
closes #7 
- This creates the base structure for the `Home View` and the folder structure for components.
- The starting `App.js` was wiped clean and given a basic starting layout
- Multiple files were cleaned up
- Folder/file structure: `components/screens/{ComponentNameFolder}/{ComponentNameFile(.js, .css, .png)}/`

**NOTE:**
Styling was not a priority with this work as the UI library has not yet been implemented.

## To Test
1. Switch to the `setup-main-menu` branch
2. Pull the changes
3. Start the metro
4. Start the app
5. Verify the home page looks correct
<img src="https://user-images.githubusercontent.com/35348015/137242871-c5753955-b696-4bf6-8fe7-78e20a6b6fd1.png" width="400">
6. Verify the buttons work
